client/app/components/group/_pickupEditCreate/pickupEditCreate.controller.js

```diff
@@ -137,4 +137,14 @@ class pickupEditCreateController {
   }
 }
 
+  noPastDateCalender() {
+  this.myDate = new Date();
```

Comment:
better call this `now`?
Also, you just use this in local scope, so you don't need to define it on `this`.